src/less/switch/base/switch.less

```diff
@@ -58,16 +54,20 @@ input.switch__control {
     z-index: 1;
 
     &[disabled] + span.switch__button {
-        opacity: 0.5;
+        opacity: 0.41;
```

Comment:
With the checkbox and radio being `.5` and `.25`, is this really `.41`? Seems kind of odd for it to not be on the same scale as the others.

Reply:
Yeah, I had to play around with it for a while. 
It's because the background color of the switch is 767676 and the disabled color is supposed to be c7c7c7.  So to get there using opacity it needs to be 0.41.  Where checkbox goes from 111820 to c7c7c7.

Reply:
Weird. Okay, I guess my comment about color+opacity applies here, too, then. Since it seems like we're jumping through hoops to get the right disabled color when the switch is "checked".

Reply:
Why not just apply the c7c7c7 color on the background icon and color the inline one. And just do away with opacity?

Reply:
Right. That's what I was getting at. Thanks for saying it succinctly.

Reply:
The current implementation used opacity.  It was just the wrong opacity for the color they were trying to achieve.  But it makes sense to use color instead.  I don't know if there was a specific reason it was done this way originally.